### PR TITLE
new.com.channels.and.personal.data

### PR DESCRIPTION
**Pull request by nodatall**

### CODE REVIEW

File: personal_data_labels.js [hunk 1]
```diff
@@ -15,6 +15,11 @@ module.exports = Object.freeze({
   homephone: 'Home Phone Number (Land Line)',
   mobilephone: 'Mobile Phone Number',
   faxnumber: 'Fax Number',
+  twitter: 'Twitter',
```
Comment (deadlyicon) [inline personal_data_labels.js]: shouldn't these be `twitter handle` and `linkedin url` etc?

File: test/mergeAccountData.spec.js [hunk 2]
```diff
@@ -88,7 +88,6 @@ describe('mergeAccountData', function(){
             communication_channels: {
               email_media: { enabled: false },
               fax_media: null,
-              social_media: { enabled: true },
```
Comment (deadlyicon) [inline test/mergeAccountData.spec.js]: why not just change this to `twitter` ?

File: personal_data_labels.js (hunk 1 above)
Comment (nodatall) [reply personal_data_labels.js]: yeah, updated

File: test/mergeAccountData.spec.js (hunk 2 above)
Comment (nodatall) [reply test/mergeAccountData.spec.js]: good point, changed

**Pull request merged**